.github/workflows/build_and_test.yml

```diff
@@ -108,6 +119,12 @@ jobs:
         script_args=()
 
 
+        DOCKER_CERTS_DIR=${{ inputs.DOCKER_CERTS_DIR }}
+        DOCKER_CERTS_UPDATE_COMMAND=${{ inputs.DOCKER_CERTS_UPDATE_COMMAND }}
+        NPROC=${{ inputs.NPROC }}
```

Comment:
If you can, you could define `nproc` as a `script_args` (`--nproc=...`), which would default (in the same way we handle the default arguments) to `nproc` in `ci_build_and_test_in_container.sh`?

Reply:
this is done...and it is apparently `--nproc ...` with no equals!